packages/eslint-config-expo/utils/core.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     ecmaVersion: 2022,
     ecmaFeatures: { impliedStrict: true, jsx: true },
   },
-  env: { es2022: true },
+  env: { es2022: true, jest: true },
```

Comment:
Instead of adding "jest" everywhere, what if we enabled this only for file paths that match Jest's [default](https://jestjs.io/docs/configuration#testmatch-arraystring): `[ "**/__tests__/**/*.[jt]s?(x)", "**/?(*.)+(spec|test).[jt]s?(x)" ]`? That way, calling `expect()` in application code is still considered a lint error.

Of course, some projects will change `testMatch` in their Jest configs. They can correspondingly override ESLint if they've also overridden Jest.

Reply:
That's a good idea! Updated.